cli/src/cli.rs

```diff
@@ -121,3 +121,11 @@ pub struct Cli {
 	#[structopt(flatten)]
 	pub run: RunCmd,
 }
+
+impl Cli {
+	/// Updates the prometheus metric prefix.
+	/// Must be called before `CliConfiguration::create_configuration()`.
+	pub fn update_prometheus_metric_prefix(&mut self, prefix: &'static str) {
+		self.run.base.prometheus_metric_prefix = prefix;
+	}
+}
```

Comment:
Not really a fan of modifying the `Cli` struct.

Reply:
This is not needed anymore as the metrics prefix has been removed in substrate.

Reply:
removed